librz/core/cmd/cmd_eval.c

```diff
@@ -53,7 +53,92 @@ static bool pal_seek(RzCore *core, RzConsPalSeekMode mode, const char *file, RzL
 	return true;
 }
 
-RZ_API bool rz_core_theme_load(RzCore *core, const char *name) {
+RZ_API bool rz_core_theme_print_metainfo(RzCore *core, RzCmdStateOutput *state, RzThemeInfo *meta_info) {
+	bool failed = false;
+
+	if (meta_info->author != NULL) {
+		rz_cons_printf("%s", meta_info->author);
+	}
+	if (meta_info->license != NULL) {
+		rz_cons_printf("%s", meta_info->license);
+	}
+	if (meta_info->targetbg != NULL) {
+		rz_cons_printf("%s", meta_info->targetbg);
+	}
+	if (meta_info->mincolordepth != NULL) {
+		rz_cons_printf("%s", meta_info->mincolordepth);
+	}
+	if (meta_info->desc != NULL) {
+		rz_cons_printf("%s", meta_info->desc);
+	}
+	if (meta_info->additionaldts != NULL) {
+		rz_cons_printf("%s", meta_info->additionaldts);
+	}
+	if (meta_info->refs != NULL) {
+		rz_cons_printf("%s", meta_info->refs);
+	}
+	return (!failed);
+}
+
+RZ_API RzThemeInfo *rz_core_theme_load_metainfo(RzCore *core, const char *name, RzCmdStateOutput *state) {
+	RzThemeInfo *meta_info = malloc(2 * sizeof(RzThemeInfo));
+	meta_info->additionaldts = NULL;
+	meta_info->refs = NULL;
+	meta_info->load_status = false;
+	if (!name || !*name) {
+		return meta_info;
+	}
+
+	char *home_themes = rz_path_home_prefix(RZ_THEMES);
+	char *system_themes = rz_path_system(RZ_THEMES);
+	char *home_file = rz_file_path_join(home_themes, name);
+	char *system_file = rz_file_path_join(system_themes, name);
+	free(system_themes);
+	free(home_themes);
+	FILE *path = rz_sys_fopen(system_file, "r");
+	if (!path) {
+		RZ_LOG_ERROR("Error in opening colorscheme file at '%s'\n", system_file);
+		return meta_info;
+	} else {
+		rz_cons_printf("Color theme details!\n");
+	}
+	bool beginflag = true;
+	char line[256];
+
+	while ((fgets(line, sizeof(line), path) != NULL) && (rz_str_cmp(line, "# ATEM\n", -1) != 0)) {
+		if (beginflag == true) {
+			if (strstr(line, "Author") != NULL) {
+				meta_info->author = strdup(line);
```

Comment:
The information should contain only author, without the `Author: `, etc
You can use `rz_str_startswith()` instead of `strstr()`